.travis.yml

```diff
@@ -23,10 +23,8 @@ install:
       cd ../protoc
       wget https://github.com/google/protobuf/releases/download/v3.2.0/protoc-3.2.0-${TRAVIS_OS_NAME}-x86_64.zip
       unzip protoc-3.2.0-${TRAVIS_OS_NAME}-x86_64.zip
-      git clone https://github.com/googleapis/googleapis.git
-      cd include/google
-      ln -s ../../googleapis/google/rpc
     )
+  - git clone https://github.com/googleapis/googleapis.git $GOPATH/src/github.com/googleapis/googleapis
```

Comment:
Maybe add a comment to point out that although this repo doesn't have any Go code in it, we need it to be accessible relative to `$GOPATH` so that .pb.go regeneration works.